src/lib/utils.sh

```diff
@@ -116,3 +116,50 @@ function send-log() {
 
   return 0
 }
+
+function sort-logs() (
+  set -euo pipefail
+
+  if [[ "$CAUR_TYPE" != 'primary' ]]; then
+    echo 'Only primary node can do this action.'
+    return 0
+  fi
+
+  local CAUR_LOG_CACHE
+  CAUR_LOG_CACHE="$(mktemp '/tmp/chaotic/logs-XXXXXXXXXX')"
+
+  if ! [[ -d "$CAUR_LOG_CACHE" ]]; then
+    echo 'It was not possible to create a temporary directory for this action.'
+    return 62 # Nico, find the next number, and put it here, I lost count already.
```

Comment:
:rofl: you're supposed to replace "62" with the proper increment of the biggest return code so far (and remove the commentary)